src/pilot/templates/dellemc-powerflex-cinder-conf.yaml

```diff
@@ -0,0 +1,17 @@
+parameter_defaults:
```

Comment:
We should use the environment files , not the extraconfig

Reply:
Don't understand your point. This file is an environment file.

Reply:
It is using "ControllerExtraConfig", it is not using the tripleo-heat-templates.
It should use - https://github.com/openstack/tripleo-heat-templates/blob/master/environments/cinder-dellemc-powerflex-config.yaml

Reply:
It's very unfortunate that we bring it up now. All the deployment has been done with ExtraConfig. Still, if we want to use the heat template that you are pointing out: 
- How do you turn on glance backend?
- In the resource registry, the CinderBackendPowerFlex resource is is calling  .../deployment/cinder/cinder-backend-powerflex-puppet.yaml which doesn't exist. There is no such file in the deployment/cinder directory. However I can see a file with similar name called cinder-backend-dellemc-vxflexos-puppet.yaml. Did you ever test it? I don't think it's gonna work.